src/testing/issue_235_regression.xspec

```diff
@@ -2,9 +2,9 @@
 <x:description 
     xmlns:x="http://www.jenitennison.com/xslt/xspec"
     xmlns:mv="http://g"
```

Comment:
Hi, @aj-stein-nist . Thanks for the quick code review. I just noticed this namespace declaration on line 4 that isn't being used. Do you want me to push another commit to remove it? It's not causing any harm; merely extraneous.

Reply:
Sorry @galtm I merged without even looking at that -- we can fix when we come back --